CONTRIBUTING.md

```diff
@@ -28,6 +28,7 @@ To run linting:
 
 ```bash
 flake8 .
+pre-commit run --all-files
```

Comment:
this requires `pre-commit` to be installed. Might be worth giving https://devenv.sh/ a thought, that's how I currently develop on this module.